projects/kit/src/lib/plugins/time-segments-stepping.ts

```diff
@@ -0,0 +1,128 @@
+import type {MaskitoPlugin} from '@maskito/core';
+import {maskitoUpdateElement} from '@maskito/core';
+
+import type {MaskitoTimeSegments} from '../types';
+
+const noop = (): void => {};
+
+export function createTimeSegmentsSteppingPlugin({
+    step,
+    fullMode,
+    timeSegmentMaxValues,
+    timeSegmentMinValues,
+}: {
+    step: number;
+    fullMode: string;
+    timeSegmentMaxValues: MaskitoTimeSegments<number>;
+    timeSegmentMinValues: MaskitoTimeSegments<number>;
+}): MaskitoPlugin {
+    const segmentsIndexes = createTimeSegmentsIndexes(fullMode);
+
+    return step <= 0
+        ? noop
+        : element => {
+              const listener = (event: KeyboardEvent): void => {
+                  if (event.key !== 'ArrowUp' && event.key !== 'ArrowDown') {
+                      return;
+                  }
+
+                  event.preventDefault();
+                  const selectionStart = element.selectionStart || 0;
+                  const selectedSegment = getSelectedSegment({
+                      segmentsIndexes,
+                      selectionStart,
+                  });
+
+                  if (!selectedSegment) {
+                      return;
+                  }
+
+                  const segmentSelection = segmentsIndexes.get(
+                      selectedSegment,
+                  ) as number[];
+
+                  const updatedValue = updateSegmentValue({
+                      selection: [
+                          segmentSelection[0],
+                          segmentSelection[segmentSelection.length - 1],
+                      ],
```

Comment:
```suggestion
                  const updatedValue = updateSegmentValue({
                      selection: segmentsIndexes.get(selectedSegment),
```

Reply:
Yep, i see that `segmentsIndexes` can includes more than 2 digits. 
Do we really required to include ALL possible indexes inside `segmentsIndexes`.

Should it just `segmentRange` (`[from: number, to:number]`) ?

Reply:
Probably, we should discuss this point tomorrow